catalog/transformer/catalog-transformer-rtf/src/main/java/ddf/catalog/transformer/output/rtf/model/ExportCategory.java

```diff
@@ -89,6 +89,7 @@ public void setAttributes(List<String> attributes) {
   @Override
   public Map<String, ExportValue> toExportMap(Metacard metacard) {
     return attributes.stream()
+        .filter(s -> metacard.getAttribute(s) != null)
```

Comment:
❓Do we want to remove attributes with null/empty values too?

Reply:
Yes. This avoids having multiple tables with no data in them as things like "Contact" are often empty.

Reply:
Just want to make sure we're on the same page - I'm asking if we need additional checks on the attribute's `getValue()` for null/empty.

Reply:
yeah, probably want to make sure it's not an empty string